### DISCUSSION TITLE
Drawing concrete and steel details

### DISCUSSION THREAD

**Discussion by ebrahimraeyat** [xcfem/xc#71] [Q&A] [answered]
Hi. I am curious that how do you draw your detailed concrete or steel plans after design? Because as you may know, FreeCAD has Reinforcement Workbench for drawing concrete detail:

[FreeCAD-Reinforcement](https://github.com/amrit3701/FreeCAD-Reinforcement)

Thanks.

**Comment by lcpt** [comment, marked as answer]
Hi @ebrahimraeyat, thank you!

We have used FreeCAD to draw detailing plans of reinforced concrete and steel structures but mostly using ad hoc Python scripts (it was about 2010). Now we are exploring the use of IFC with FreeCAD to "upgrade" our workflow. I didn't know this add-on, but we will certainly use it as soon as possible.

Thanks again.

**Comment by ebrahimraeyat** [comment]
Thanks for your quick reply. Related link:

[Extended functionality of Rebar Addon](https://forum.freecadweb.org/viewtopic.php?f=23&t=44580)

[FreeCAD-Reinforcement](https://github.com/SurajDadral/FreeCAD-Reinforcement)
